build.gradle

```diff
@@ -193,8 +193,6 @@ configurations {
 
 dependencies {
     implementation project(path: ":${rootProject.name}-spi", configuration: 'shadow')
-    implementation group: 'com.google.guava', name: 'guava', version:'32.1.3-jre'
```

Comment:
I think the whole section above this can go too:

```
configurations {
    all {
        resolutionStrategy {
            force 'com.google.guava:guava:32.1.3-jre'
        }
    }
}
```

Also what about the section below?

```
    //spotless
    implementation('com.google.googlejavaformat:google-java-format:1.26.0') {
        exclude group: 'com.google.guava'
    }
```

Reply:
Good catch, I am removing them as well.